eng/scripts/InstallVisualStudio.ps1

```diff
@@ -35,7 +35,7 @@
 #>
 param(
     [ValidateSet('BuildTools','Community', 'Professional', 'Enterprise')]
-    [string]$Edition = 'Enterprise',
+    [string]$Edition = 'Community',
```

Comment:
This is a parameter and should not need to be changed. Please revert this file.

Reply:
yes sorry i pushed all untracked files by mistake; I have opened new pull request now with everything in order, sorry about that